### ISSUE TITLE
Honduras Planificación

### ISSUE THREAD

**Issue by Caballero-Arepa** [Caballero-Arepa/Latin-American_Civs#64]
###    **Banana Capital**
[stats] from Bananas tiles
### **Banana Hacienda**
Creates a Platanera improvement on a tile
### _Platanera_
Improvement that provides some stats and provides Banana resource.
### **Lenca Chieftain**
Replaces Composite Bowman
8 DEF, 14 DMG, 2 RANGE, 2 MOVEMENT Compared to the original 7 DEF, 11DMG, 2 RANGE, 2 MOVEMENT.
Can go through impassable tiles
Cover 1

_For the 3&4 Uniques:_
### **Lancetilla Botanic Garden**
The second biggest in the world, it's a national wonder that places a tile with high yields
### **Raining Fish**
Honestly idk XD
It can't be build next to coast, provides fish in this city.

**Comment by Caballero-Arepa** [comment]
El Jardin Botanico Lancetilla es el segundo más grande del mundo.
Honduras es la capital del banano
Lluvia de peces de Lloro

**Comment by BrokenCartridge** [comment]
English:
The name of the leader could be: Francisco Morazán of Honduras
(He dominated the political and military scene of Central America until his execution in 1842. In the political arena, Francisco Morazán was recognized as a visionary and great thinker, as he attempted to transform Central America into one large and progressive nation.) 
Español:
El nombre del líder podría ser: Francisco Morazán de Honduras (Dominó el escenario político y militar de Centroamérica hasta su ejecución en 1842. En el ámbito político, Francisco Morazán fue reconocido como un visionario y gran pensador, pues intentó transformar América Central en una nación grande y progresista).

**Comment by BrokenCartridge** [comment]
English:
The starting bias would be Jungle since thats where bananas can be and Honduras is the banana capital.
Español:
El sesgo inicial sería jungla ya que ahí es donde pueden estar los bananos y Honduras es la capital del banano.

**Comment by BrokenCartridge** [comment]
> ### **Lempira**
> A hero like unit, kinda strong, but you can only make one. Once it's dead, it gives a massive Culture boost, but can't be built again.

I like the idea of adding Lempira but maybe we should tweak it a bit more since having a single use troop is a major downside. 

Instead of just Lempira, we have 'Lenca Cheiftain' which was his title. 

Lempira used arrows with obsidian darts like other Lenca's as weapons (even has a statue with him wearing a quiver) so they would replace Composite Bowmans 

One of its unique perks could be being able to move onto mountains since Lempira's name means 'Lord of the Mountains'. 

Another potential unique perk is being immune to or having a big defense against its own unit type, since Lempira believed that wearing the armor of Spaniards made him Immune to Spaniard weapons which were mainly muskets 

Edit: We could do what the Mongol has in the game and replace the Great General with Lempira!


TL;DR:

 Instead of Lempira as a unique troop instead have:
 'Lenca Cheiftain' (Replaces Composite Bowman)
8 DEF, 14 DMG, 2 RANGE, 2 MOVEMENT Compared to the original 7 DEF, 11DMG, 2 RANGE, 2 MOVEMENT.

Potential Free Promotions: 
'Lord of Mountains'
Can go through Impassable Tiles 
'Enemy Armor'
Immune to (or high defense against) ranged attacks

**Comment by Caballero-Arepa** [comment]
Oh yeah, that's a neat idea. I'll go for that instead.
